sql/conseil.sql

```diff
@@ -148,10 +169,20 @@ CREATE TABLE public.blocks (
 
 --
 -- TOC entry 186 (class 1259 OID 99628)
--- Name: delegates; Type: TABLE; Schema: public; Owner: -
+-- Name: delegated_contracts; Type: TABLE; Schema: tezos; Owner: -
+--
+
+CREATE TABLE tezos.delegated_contracts (
```

Comment:
This table has been removed in the proto5 branch and should stay removed.

Reply:
my mistake, fixed